### PR TITLE
Add sort function that will sort the games based on the last updated

### LINKED ISSUE #207: Games chronologisch ordenen
Branch name: `feature/games-chronologisch-ordenen`
Blocked by: https://github.com/iSirThijs/jungle-gym-api/issues/9

**Omschrijving:**
Nieuwste komt bovenaan in de lijst

### PR DESCRIPTION
**Pull request by jochemvogel**
# Description

Closes #207 

De game die als laatst bewerkt (updated) is (DUS NIET LAATST TOEGEVOEGD) komt bovenaan te staan. Nieuwe games komen daardoor automatisch bovenaan in het speloverzicht te staan.

## Type of change

Please select the option that is relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] All the required documentation is added.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.

# Additional information

_[Provided extra information if necessary]_

### CODE REVIEW

File: src/lib/GameList/GameList.svelte [hunk 1]
```diff
@@ -8,6 +8,11 @@
 	export let query: string;
 	export let offline: boolean;
 
+	// Sort the array with the last updated games as first
```
Comment (jochemvogel) [inline src/lib/GameList/GameList.svelte]:
```suggestion
	// Sort the array with the last updated game first
```

**Comment by iamDistractech** [comment]
Nu we de API dus op een andere manier hebben, moet ik die `updatedAt` wel doorsluisden van Strapi aan de output van mijn API toch? Also dan moet de `typing` van `Game.d.ts` aangepast worden..

**Review by sreen020 — APPROVED** [review]

**Comment by jochemvogel** [comment]
Ik ga hier morgen naar kijken @iSirThijs. Daarna heb ik antwoord op je vraag.

**Comment by jochemvogel** [comment]
Blocked: https://github.com/iSirThijs/jungle-gym-api/issues/9

**Pull request merged**